### ISSUE TITLE
How to run multiple nodes in different machines

### ISSUE THREAD

**Issue by ashish10677** [kurtosis-tech/ethereum-package#621]
# Description

I am running eth testnet using the example: https://docs.kurtosis.com/how-to-local-eth-testnet. I want to run other Geth full nodes on other VMs. Is there a tutorial or easy way to do this?

**Comment by barnabasbusa** [comment]
You can only use docker or kubernetes as a backend. You can't target specifically a different node. If you want to target multiple nodes, I'd recommend you to take a look at setting up some kubernetes cluster and target that with kurtosis.

**Comment by ashish10677** [comment]
> You can only use docker or kubernetes as a backend. You can't target specifically a different node. If you want to target multiple nodes, I'd recommend you to take a look at setting up some kubernetes cluster and target that with kurtosis.

Just confirming, I am running one node in a private network using kurtosis example. I want to run a few other nodes in seperate VMs, to create a private blockchain. That's not possible?

**Comment by barnabasbusa** [comment]
No, they would need to be part of the same cluster in order for you to be able to target them.
